src/polar/Pie.tsx

```diff
@@ -340,22 +340,36 @@ const getTextAnchor = (x: number, cx: number) => {
   return 'middle';
 };
 
+const getOuterRadius = (
+  dataPoint: any,
+  outerRadius?: number | string | ((element: any) => number),
```

Comment:
Nit: for consistent interface this should be updated too

But it's not outward facing so all good